client/src/pages/Login.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+
+import { Button } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+import { Route, Link } from "react-router-dom";
```

Comment:
Route is defined but never used - make sure to address the warnings in the app (you can remove Ping, it's not used anymore)